expression/builtin_time_vec.go

```diff
@@ -1195,11 +1284,27 @@ func (b *builtinCurrentTime1ArgSig) vecEvalDuration(input *chunk.Chunk, result *
 }
 
 func (b *builtinUTCTimestampWithoutArgSig) vectorized() bool {
-	return false
+	return true
 }
 
+// vecEvalTime evals UTC_TIMESTAMP().
+// See https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_utc-timestamp
 func (b *builtinUTCTimestampWithoutArgSig) vecEvalTime(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	nowTs, err := getStmtTimestamp(b.ctx)
+	if err != nil {
+		return err
+	}
+	res, err := convertTimeToMysqlTime(nowTs.UTC(), types.DefaultFsp, types.ModeHalfEven)
+	if err != nil {
+		return err
+	}
```

Comment:
How about just calling `evalUTCTimestampWithFsp`.

Reply:
I will fix it.